### ISSUE TITLE
Should we build noarch Python packages?

### ISSUE THREAD

**Issue by jeanconn** [sot/skare3#5]
https://www.anaconda.com/blog/developer-blog/condas-new-noarch-packages/

**Comment by jeanconn** [comment]
While this increases complexity, I think it might save enough on maintenance to be worth it.

**Comment by taldcroft** [comment]
+100.  Most of our packages should be noarch so this is a huge win!

**Comment by taldcroft** [comment]
Can't imagine how that could work for Windows for those...

**Comment by jeanconn** [comment]
Unless we have a Windows build/test process/machine, I'm not going to have much of a clue about Windows.  I think this flag will be relatively easy to turn off (and go back to building individual arch packages) if we need to provide recipes that will work to build from Windows.

**Comment by jeanconn** [comment]
Ah good, when I tried to build Chandra.Time noarch I got a very reasonable error.  So I should be able to see which are noarch-possible on build:
```
[noarch_python] Error: Binary library or executable found: lib/python3.6/site-packages/Chandra.Time-3.20.1-py3.6-linux-x86_64.egg/Chandra/Time/_axTime3.cpython-36m-x86_64-linux-gnu.so
```

**Comment by taldcroft** [comment]
Closed by #7
